link/src/test/java/com/stripe/android/link/serialization/PopupPayloadTest.kt

```diff
@@ -58,4 +111,16 @@ internal class PopupPayloadTest {
             "link_passthrough_mode_enabled" to true,
         ),
     )
+
+    private fun getContext(locale: Locale): Context {
```

Comment:
this seems useful! Do we have test utils or anything like that? It'd be nice to have an easy way to find functions like this if I ever wanted to use one. Not necessary for this PR just an idea :)

Reply:
We could add a test util for this as part of the new testing module @tjclawson-stripe will be adding.